src/v/cloud_storage/topic_manifest.h

```diff
@@ -52,6 +55,11 @@ class topic_manifest final : public base_manifest {
     /// Change topic-manifest revision
     void set_revision(model::initial_revision_id id) noexcept { _rev = id; }
 
+    std::optional<cluster::topic_configuration>
```

Comment:
looks like you can return by `const&` to avoid a copy

Reply:
done